managed/ui/src/components/universes/UniverseDetail/compounds/UniverseHealthCheckList.js

```diff
@@ -225,7 +224,7 @@ const detailsFormatter = (cell, row) => {
 function prepareData(data) {
   return data.map((timeDataJson) => {
     const timeData = JSON.parse(timeDataJson);
-    const timestampMoment = moment(timeData.timestamp);
+    const timestampMoment = momentTimezone.utc(timeData.timestamp).local();
```

Comment:
@jaydeepkumara are you sure about `momentTimezone`?
it seems `moment.utc()` is what we need
https://momentjs.com/docs/#/parsing/utc/

Reply:
changed to a moment. Both libraries support UTC fun. The earlier moment had given an error so had used the moment timezone but I changed the import statement and it worked now.